app/frontend/src/registry/components/people/Notes.vue

```diff
@@ -50,16 +55,88 @@
               </b-btn>
             </div>
           </b-modal>
+          <b-modal
+              v-model="confirmDeleteModal"
+              centered
+              title="Confirm Deletion"
+              @shown="focusDeleteModal"
+              :return-focus="$refs.noteInputCancelBtn">
+              <p>Are you sure you want to delete this note?</p>
+              <div v-if="activeNote" >
+                <p class="font-weight-bold">"{{activeNote.note}}"</p>
```

Comment:
Same as line 119 - should add `word-break: break-all` styling to this element to wrap any very long, unbroken text.
<img width="922" alt="Screenshot 2024-03-28 at 12 48 00 PM" src="https://github.com/bcgov/gwells/assets/62899351/3e4e5d65-b523-4da9-b155-cc115a70794c">

Reply:
![image](https://github.com/bcgov/gwells/assets/62873746/c05f1a51-fb91-4bda-be5c-de78d7322321)
Implemented